models/address_book.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'entry'
+
+class AddressBook
+  attr_accessor :entries
+
+  def initialize
+    @entries = []
+  end
+
+  def add_entry(name, phone_number, email)
+    index = 0
+    @entries.each do |entry|
+      if name < entry.name
+        break
+      end
+      index += 1
+    end
+
+    @entries.insert(index, Entry.new(name, phone_number, email))
+  end
+# Code for assignment: remove_entry method.
+  def remove_entry(name, phone_number, email)
+    delete_entryj = nil
```

Comment:
Actually, it doesn't. I removed it...I can't remember if I placed it there so I wouldn't forget to set it, but I actually put it in twice.